CommunicationLayer/CommDeviceControl/UdpMessageForwarder.cpp

```diff
@@ -26,27 +26,39 @@
 #include <QByteArray>
 #include <QDebug>
 #include <QHostAddress>
+#include <QTextCodec>
+
 
 #include "InfrastructureLayer/Settings/I_Settings.h"
 #include "UdpMessageForwarder.h"
 
+namespace
+{
+    const QString EXCHANGE_NAME = QString("amq.direct");
```

Comment:
I believe that by default there is an exchange on the server created of type `amq.direct` that also happens to be named `amq.direct`

I am unsure whether or not I am correct as my memory may be failing me, but if this is the case can I continue using default exchanges or should I declare a new one as you stated (speaking of type fanout now)